lidlplus/__main__.py

```diff
@@ -119,6 +123,24 @@ def print_tickets(args):
         tickets = lidl_plus.ticket(lidl_plus.tickets()[0]["id"])
     print(json.dumps(tickets, indent=4))
 
+def activate_coupons(args):
+    """Activate all available coupons"""
+    lidl_plus = lidl_plus_login(args)
+    if not args.get("all"):
+        return
+    i = 0
+    for section in lidl_plus.coupons()["sections"]:
+        for coupon in section["coupons"]:
+            if coupon["isActivated"]:
+                continue
+            if datetime.fromisoformat(coupon["startValidityDate"]).timestamp() > datetime.now().timestamp():
+                continue
+            if datetime.fromisoformat(coupon["endValidityDate"]).timestamp() < datetime.now().timestamp():
+                continue
```

Comment:
It is necessary to use the .timestamp() function here? datetime objects can be compared directly :slightly_smiling_face: 

```suggestion
            if datetime.fromisoformat(coupon["startValidityDate"]) > datetime.now():
                continue
            if datetime.fromisoformat(coupon["endValidityDate"]) < datetime.now():
                continue
```